kubernetes/cray-tftpd-ipxe/values.yaml

```diff
@@ -53,6 +53,11 @@ cray-service:
   replicaCount: 3
   containers:
     - name: cray-tftp
+      securityContext:
+        runAsUser: 0
```

Comment:
Does this need to run as root?  Default should not be, so adding this escalates the security privileges?

Reply:
I followed what was outlined in this ticket here https://jira-pro.its.hpecorp.net:8443/browse/CASMCMS-7942. 
It seems that without root this container fails with a permissions issue.